compiler/rustc_expand/src/expand.rs

```diff
@@ -1706,9 +1733,29 @@ impl<'a, 'b> InvocationCollector<'a, 'b> {
             return match self.take_first_attr(&mut node) {
                 Some((attr, pos, derives)) => match attr.name_or_empty() {
                     sym::cfg => {
-                        if self.expand_cfg_true(&mut node, attr, pos) {
+                        let (res, meta_item) = self.expand_cfg_true(&mut node, attr, pos);
+                        if res {
                             continue;
                         }
+                        debug!("Stripping out node");
```

Comment:
Could you remove all the added debug printing here and in other places?

Reply:
Just the `debug!`s or also the `instrument` in the other places? I found them quite useful for debugging so they might be useful in the future as well.

Reply:
>I found them quite useful for debugging

I don't doubt this part, but

> so they might be useful in the future as well.

is almost never true.

Reply:
fair. They're pretty easy to add if needed again. I'll remove them at the end after the PR is finished.